extension/controller/backgroundScript.js

```diff
@@ -326,6 +335,38 @@ const messageListener = function(message, sender) {
             translatingAsBrowse: message.translatingAsBrowse
           });
           break;
+        case "errorCollectionConsent":
+          browser.storage.local.set({ errorCollectionConsent: message.consent });
+          if (message.consent) {
+            if (!Sentry.getCurrentHub().getClient()) {
+              Sentry.init({
```

Comment:
what will happen if we init it the second time?

Reply:
That won't happen. `if (!Sentry.getCurrentHub().getClient()) ` will safegaurd it